tasks/salesforce_robot_library_base.py

```diff
@@ -0,0 +1,58 @@
+import logging
+import os
+import pprint
+
+from cumulusci.core.config import TaskConfig
+from robot.libraries.BuiltIn import BuiltIn
+from cumulusci.tasks.apex.anon import AnonymousApexTask
+
+from tasks.npsp_batch import BatchApexWait
+
+
+class SalesforceRobotLibraryBase(object):
```

Comment:
This is not a task. Maybe it should go in robot/Cumulus/resources instead of tasks.

Reply:
Can't any folder be treated as a package in Python 3?

Reply:
It may be a problem specific to the way that Robot loads .py files.

ImportError: attempted relative import with no known parent package

I can clean this up by moving SalesforceBase to the CCI repro (which is my plan anyhow) but I'd rather do that asynchronously instead of waiting to merge this PR. Once its been deployed over there I can delete it here, similar to some of the other files.

Reply:
I'm okay with that.